actions/compute-payload.go

```diff
@@ -362,3 +384,143 @@ func ComputeEAD(damages []float64, freq []float64) float64 {
 	}
 	return eadT
 }
+
+// results writer for spatial data to psql database
+type psqlResultsWriter struct {
```

Comment:
all of this looks reasonable, my strong suggestion is that the entire struct and behaviors that implement the results writer interface should be moved out of the actions module into a new module called "resultswriters" (much like hazardproviders) to allow for additional implementations to be added in a logical place in the future.